packages/api/src/hooks/useIsEuRegion.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import useLandingCompany from './useLandingCompany';
+
+const useIsEuRegion = () => {
+    const { data: landing_company, ...rest } = useLandingCompany();
+
+    const isEuRegion = useMemo(() => {
+        if (!landing_company) return false;
```

Comment:
```suggestion
        if (!landing_company) return;
```

Reply:
returning `false` as default according to the original code from `client-store`
<img width="712" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/d64d262e-4b8c-4282-85c1-8676d8475e46">
@thisyahlen-deriv